test/CMakeLists.txt

```diff
@@ -57,6 +56,7 @@ add_custom_command(OUTPUT ${CMAKE_BINARY_DIR}/test/data/tz.sqlite
   COMMENT "Building tz.sqlite..."
   WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
 )
+add_custom_target(build_timezones DEPENDS ${CMAKE_BINARY_DIR}/test/data/tz.sqlite)
```

Comment:
by doing this its much easier to make other targets depend on the tz.sqlite that is created even the gurka tests